### PR TITLE
Tap on notification

### PR DESCRIPTION
**Pull request by gabrielbazan7**
fix : #4154

### CODE REVIEW

**Comment by gabrielbazan7** [comment]
There is a plugin error in version 1.2.3 for IOS devices in this feature.

**Comment by gabrielbazan7** [comment]
Updated

File: cordova/build.sh [hunk 1]
```diff
@@ -93,6 +93,16 @@ if [ ! -d $PROJECT ]; then
     checkOK
   fi
 
+  if [ $CURRENT_OS == "IOS" ]; then
+    cordova plugin add phonegap-plugin-push --variable SENDER_ID="XXXXXXX"
```
Comment (matiu) [inline cordova/build.sh]: is XXXXXX ok??

Comment (gabrielbazan7) [inline cordova/build.sh]: Sender id maps to the project number in the Google Developer Console. So, as we are using for only IOS application we can put in anything for this value.

Comment (cmgustavo) [inline cordova/build.sh]: For iOS you set the latest version of the plugin, but for Android your are using an specific version (1.2.3). Is that ok too?

Comment (gabrielbazan7) [inline cordova/build.sh]: Yes

**Comment by gabrielbazan7** [comment]
Moved to v1.12 #4185

**Pull request closed without merging**